### ISSUE TITLE
Configured but not available extension doesn't show an error

### ISSUE THREAD

**Issue by offa** [winsw/winsw#666]
A configured extension that's not found doesn't show or log any error message. This happens eg. if the extension is not available, it's class is incorrect, or in the current version, contains a case missmatch (#662).

The installation and starting works fine, but starting crashes without any visible reason. There's neither a console output nor a log entry. Only the windows event log gives a hint:

```
WinSW.Extensions.ExtensionException: should-fail: Cannot load the class by name: not.existing ---> WinSW.Extensions.ExtensionException: should-fail: Class not.existing does not exist
```

## Steps to reproduce

Create a minimal service (eg. `test.exe`) with a configuration that references a not existing class.

```xml
<service>
    <id>example-service</id>
    <name>Example service</name>
    <description>Example.</description>
    <executable>dir</executable>

    <extensions>
        <extension enabled="true" id="should-fail" className="not existing"></extension>
    </extensions>
</service>
```

1. `test.exe install` --> Ok
1. `test.exe start` -->  Ok, but service crashed without notice

## Environment

* WinSW version: 2.10.1
* WinSW package type: .NET 4.6.1
* Windows version: 10
* Wrapped executable and version: any

## Possible Solution

Logging of the error, preferably wrapper log and console.

**Comment by NextTurn** [comment]
The underlying issue is that all errors during service starting don't show up in log files in WinSW 2.

**Comment by oleg-nenashev** [comment]
The release is out: https://github.com/winsw/winsw/releases/tag/v2.10.2 . We still need to fix logging

**Comment by NextTurn** [comment]
File logging has been fixed in #736. I doubt if console logging can be implemented only with WMI in this case because the `StartService` method is a one-shot action and the `Started` property is true for both "starting" and "running" states so you don't know when the service starts successfully or fails.

**Comment by nxtn** [comment]
At the moment errors produced by the service at runtime are not sent to the console. That would be a separate feature update.
